galleries.py

```
@@ -464,15 +492,18 @@ def _label(target, page, pages):
+class FixIndent(yaml.Dumper):
+
+    def increase_indent(self, flow=False, indentless=False):
+        return super().increase_indent(flow, False)
 
 ################################################################################
 # Internal function to write a full set of browse pages
 ################################################################################
 
-def _gallery(fileroot, product_ids, catalog, links,
-             separators=('|','|'), linebreaks=True, bottom_nav=20):
+def _gallery(fileroot, product_ids, catalog, links, resolve_urls=False):
```

Ruff (0.15.2)

[warning] Missing return type annotation for private function `_gallery`

Add return type annotation: `None`

(ANN202)